app.py

```diff
@@ -156,7 +203,74 @@ def show_column(dstats, display_list, show_perplexities, column_id=""):
     """
 
     # start showing stuff
-    st_utils.expander_header(dstats, DATASET_NAME_TO_DICT)
+
+
+    #title_str = f"### Showing{column_id}: {dstats.dset_name} - {dstats.dset_config} - {dstats.split_name} - {'-'.join(dstats.text_field)}"
+    #st.markdown(title_str)
+    tab1, tab2, tab3, tab4, tab5, tab6, tab7,tab8 = st.tabs(["Dataset Description", "General Text Statistics", "Label Distribution", "Text Lengths","Text Duplicates","Text Perplexities","nPMI","Zipfs Law Fit"])#,"Text Embedding Clustering"])
```

Comment:
Another option would be to have the `with tab` call be in streamlit_utils.py (I would prefer this, since the tabs are specific to streamlit), where -- if I'm following your changes correctly -- it would simply replace the `with st.expander` logic that's currently there.

Reply:
yes that makes sense, is the with tab in the newest merge in main, I'm not seeing it in the streamlit_utils.py? or is you recommendation to replace the with expander, with, with tab?

Reply:
Yeah, recommendation to replace the "with expander" with "with tab", moving it from app.py into streamlit_utils.py =)
(assuming that that is a valid way to do it!)

Reply:
the tabs initialisation line 210 kinda needs to stay, incorporating` st.tabs` into `display_list` creates vertical tabs,
 however, the proceeding lines ' `with tabsx:`'  have been incorporated into the loop in `show_column`